juspay/hyperswitch
# chore(env): add ttl as env variable

## Type of Change


## Description

Add TTL for KV as environment variable.

### Additional Changes




## Motivation and Context

Add TTL for KV as environment variable.

## How did you test it?

- Run the application 
- Check the startup config debug log
<img width="1728" alt="Screenshot 2023-10-20 at 4 44 53 PM" src="https://github.com/juspay/hyperswitch/assets/43412619/cbcc60d5-423c-4349-96b7-e38ed3e61937">


## Checklist
